test/functional/apk-utils-e2e-specs.js

```diff
@@ -78,6 +78,21 @@ describe('apk utils', function () {
       await assertPackageAndActivity();
 
     });
+
+    it('should be able to start with an intent and no activity', async function () {
+      await adb.install(contactManagerPath);
+      await adb.startApp({
+        pkg: 'com.example.android.contactmanager',
+        category: 'android.intent.category.DEFAULT',
```

Comment:
if category is required, should we also validate it as part of this case?

Reply:
Hmm, I'm not sure.I don't think it's required technically (I know I said that in the description though!), but it was required for my use case.

It will run etc. without an explicit category but I think if you don't provide one, it uses the default one `android.intent.category.LAUNCHER` - I *think* that's set in `appium` main. 

It might be clearer/less coupled in the test to leave out the category? I just tried it locally and it still works so I'm tempted to take it out.

Reply:
Yeah, I'd consider it a better test if it didn't have other stuff that potentially alters whether or not the tested functionality works.

Reply:
Totally agree; I wouldn't have left it in if I'd realised beforehand. Removed now, thanks!